multiscale_spatial_image.py

```diff
@@ -135,7 +135,7 @@ class Methods(Enum):
     XARRAY_COARSEN = "xarray.DataArray.coarsen"
     ITK_BIN_SHRINK = "itk.bin_shrink_image_filter"
     ITK_GAUSSIAN = "itk.discrete_gaussian_image_filter"
-    DASK_GAUSSIAN = "dask_image.ndfilters.gaussian_filter"
+    DASK_IMAGE_GAUSSIAN = "dask_image.ndfilters.gaussian_filter"
```

Comment:
I am less familiar with the dask ecosystem, is `dask` versus `dask_image` an important distinction to prevent confusion? Per my understanding `dask` does not itself implement a gaussian filter. Not opposed to the change, just want to understand the mindset.

Reply:
Yes, this is an important distinction.  `dask` is general. `dask-image` is a one package that provides some image-processing functions built on `dask`. But you can also use itk, scikit-image, etc. etc.

Reply:
Got it, thanks for the clarification!